packages/mui-joy/src/Select/Select.test.tsx

```diff
@@ -312,24 +312,12 @@ describe('Joy <Select />', () => {
       expect(getByRole('combobox')).not.to.have.attribute('aria-disabled');
     });
 
-    it('indicates that activating the button displays a listbox', () => {
```

Comment:
According to [the ARIA spec](https://w3c.github.io/aria/#combobox), `aria-haspopup="listbox"` is applied implicitly when `role="combobox"`, so there's no need to add it manually.